src/components/Stat.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+type StatProps = {
+  children: string;
+  content: string;
+  className?: string;
+};
+
+const Stat = ({ children, content, className }: StatProps) => {
+  return (
+    <div
+      className={`text-white flex justify-center align-center flex-col gap-4 mx-5 md:w-56 ${className}`}
```

Comment:
Nit: Think the theme uses the text-color-primary-text or something

Reply:
Could be worth changing the theme definition to use text-white instead, definitely makes more sense imo.

Reply:
Chnaged to basic colours